core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -75,54 +89,50 @@ var PreferencesPage = function() {
   };
 
   this.editUserBio = async function(bio) {
-    await userBioElement.sendKeys(bio);
-    await navBar.click();
-    await preferencesLink.click();
-  };
-
-  this.get = async function() {
-    await browser.get(USER_PREFERENCES_URL);
-    await waitFor.pageToFullyLoad();
+    await action.sendKeys('User bio field', userBioElement, bio);
+    await saveNewChanges('User Bio');
```

Comment:
We're no longer clicking `preferencesLink` here. Why don't we need to?

Reply:
We need just one click to trigger the digest cycle and that's being done by navbar click*